config/software/elasticsearch.rb

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright 2016 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name "elasticsearch"
+default_version "5.4.1"
+
+dependency "server-jre"
+
+license "Apache-2.0"
+license_file "LICENSE.txt"
+
+source url: "https://download.elastic.co/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/#{version}/elasticsearch-#{version}.tar.gz"
+relative_path "elasticsearch-#{version}"
+
+version "2.3.1" do
+  source sha1: "387c5f045843339d486203b0048ff8911e9c8c54"
```

Comment:
Is there a sha256 available? (if not that's fine - we just may as well set the precedent at the beginning if we can!)